build.Dockerfile

```diff
@@ -1,11 +1,8 @@
-FROM golang:1.8
+FROM golang:1.9-alpine
```

Comment:
What gives you confidence the next Go release will be out way before Traefik 1.4? Go 1.9 is only at the first release candidate, and Traefik is going to ship its first within the next few days presumably.